src/nft/components/collection/ActivityCells.tsx

```diff
@@ -56,6 +56,15 @@ const AddressLink = styled(ExternalLink)`
   }
 `
 
+const isPurchasableOrder = (orderStatus?: OrderStatus, marketplace?: string): boolean => {
+  if (!marketplace || !orderStatus) return false
+  const purchasableMarkets = Object.keys(NftMarketplace).map((market) => market.toLowerCase())
+
+  const validOrder = orderStatus === OrderStatus.Valid
+  const isPurchasableMarket = purchasableMarkets.includes(marketplace.toLowerCase())
+  return validOrder && isPurchasableMarket
+}
+
 const formatListingStatus = (status: OrderStatus): string => {
```

Comment:
I don't think these are good to unit test because they are not exported functions, so in order to unit test they would have to be exported only for testing purposes which there have been discussions about not doing

Reply:
I am adding snapshot tests though because I think that is very beneficial